### PR TITLE
[ADD] Add new module invoice_picking_address_template

### PR DESCRIPTION
**Pull request by hurrinico**

### CODE REVIEW

File: invoice_report_picking_address/README.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Invoice Report Picking Address
+==============================
+
+Adds delivery address on account invoice template
```
Comment (eLBati) [inline invoice_report_picking_address/README.rst]: I would replace ```template``` with ```report```

**Comment by eLBati** [comment]
I think module https://github.com/OCA/account-invoice-reporting/tree/7.0/account_invoice_delivery_address should be ported to 8.0 instead of creating this one

**Comment by eLBati** [comment]
@hurrinico what do you think?

**Comment by eLBati** [comment]
I noticed the old module had to change RML report and used a different approach about `partner_shipping_id` field.
I am :+1: with this module, but I would name it `account_invoice_picking_address` and add the dependency on sale_stock, adding the ability to retrieve the address from SO

**Comment by tafaRU** [comment]
@hurrinico, the git author of you commit seems unknown, could you please check it?

**Comment by eLBati** [comment]
Closing this in favour of https://github.com/OCA/account-invoice-reporting/pull/40

**Pull request closed without merging**